src/core/service.c

```diff
@@ -246,6 +256,20 @@ static void service_reset_watchdog(Service *s) {
         service_start_watchdog(s);
 }
 
+static void service_reset_watchdog_timeout(Service *s, usec_t watchdog_override_usec) {
+        assert(s);
+
+        if (watchdog_override_usec == 0)
+                return ;
```

Comment:
I thought that changed watchdog timeout for 0 sec is meaningless,
but now I think I need to change it.
I will add new variable watchdog_override_enable, so watchdog_override_usec does not need having range.